dimagi/utils/couch/migration.py

```diff
@@ -135,10 +136,10 @@ def _migration_get_couch_model_class(cls):
         raise NotImplementedError()
 
     def _migration_get_couch_object(self):
-        if not self.couch_id:
+        if not getattr(self, self.couch_id_name, None):
```

Comment:
agree, i think i would have preferred a method called `_migration_get_couch_id_property_name` on both classes to do this, which defaults to 'couch_id'